packages/loot-core/src/platform/server/sqlite/index.web.test.ts

```diff
@@ -27,8 +38,7 @@ describe('Web sqlite', () => {
 
     let rows = runQuery(db, 'SELECT * FROM numbers', null, true);
     expect(rows.length).toBe(1);
-    // @ts-expect-error Property 'number' does not exist on type 'unknown'
-    expect(rows[0].number).toBe(4);
+    expect((rows[0] as NumberRowType).number).toBe(4);
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏manual type coercions _hide_ the type issue rather than address it. Furthermore: they can and do often lead to subtle, hard-to-catch bugs.

If we cannot patch this properly (i.e. by explicitly stating the `runQuery` return type), then the previous solution of using `@ts-expect-error` is actually better.

Reply:
Oh. I disagree that expect-error is better, adding linter exceptions creates a culture of accepting the problem, instead of fixing it at the level you are able, as well as having other issues. But, I also understand it's not my call, and code consistency matters, so I'll revert those changes.